en/gallery.html

```diff
@@ -70,19 +60,27 @@
       <!-- Gallery -->
 
       <div align="center">
+
+        <h2> Download pictures </h2>
+
+  <img src="/img/soprano.webp" alt="Picture showing Cesar playing with a Soprano Sax">
+        <br>
+  <a href="/img/soprano.webp" download="cesar_joaniquet_soprano.webp" class="download-link">Click to download picture</a>  

```

Comment:
proof of concept: High quality images with a download button, so press may download it themselves without needing to respond e-mails all the time.